section-05-production-model-package/regression_model/config.yml

```diff
@@ -0,0 +1,172 @@
+# Package Overview
+package_name: regression_model
```

Comment:
are constants not supposed to be written in capitals?

Reply:
This is yaml, so we adhere to yaml conventions.